addons/kommander/1.1.0/kommander-57.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-57"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0"
```

Comment:
Are we going to retag? or should this be `1.1.1`?

Reply:
retag!